.github/workflows/python-package.yml

```diff
@@ -23,11 +23,26 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with flake8
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        pip install flake8
+        flake8 climpred
+    - name: Format with black
+      run: |
+        pip install black==19.10b0
+        black --check climpred
+    - name: Format with isort
+      run: |
+        pip install isort
+        isort --recursive --check-only climpred
+    - name: Run tests
+      run: |
+        pip install pytest pytest-cov pytest-sugar pytest-tldr pytest-lazy-fixture
+        pip install coveralls
+        pip install netcdf4
+        pip install ipython
+        python -m pip install "dask[array]" --upgrade
```

Comment:
so you dont want to create an env from conda but rather pip install everything?

Reply:
Just trying to figure out the minimum packages needed for testing. Then will create a testing env. The problem is then we still have to install miniconda, etc. which adds so much time to the testing suite I feel. Well I can look at the total time with the comprehensive pip and then do it with conda and compare.

Reply:
so maybe we have a minimum required to pass all tests excluding the pytest-tldr nice to have packages